config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  namespace :api do
+    namespace :v1 do
+      root "/health#show"
```

Comment:
It is kind of repetitive though 🤔 And there should be only one `root`, no? Will this be removed?

Reply:
Since we don't want `baseUrl/api/v1` to break, I've added a root here 🙏